client/chat/channel.jsx

```diff
@@ -71,6 +71,18 @@ const UserSublist = styled.ul`
   list-style-type: none;
   margin: 0;
   padding: 0;
+
+  ${props => {
+    if (props.sectionTitle === 'Offline') {
+      return `
+        color: ${colorTextFaint};
```

Comment:
`colorTextFaint` should be reserved for things that don't need to be readable as it doesn't generally meet contrast guidelines (basically, only the text of disabled things). The lowest you should go for stuff that does need to be readable is `colorTextSecondary`